health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -263,16 +290,38 @@
-function processErrorDataForTargets(request: any, createAndSearchConfig: any, workbook: any, sheetName: any) {
+function processErrorDataForEachSheets(request: any, createAndSearchConfig: any, workbook: any, sheetName: any) {
     const desiredSheet = workbook.getWorksheet(sheetName);
     const columns: any = findColumns(desiredSheet);
     const statusColumn = columns.statusColumn;
     const errorDetailsColumn = columns.errorDetailsColumn;
+    const userNameAndPassword = request?.body?.userNameAndPassword;
 
-    const errorData = request.body.sheetErrorDetails.filter((error: any) => error.sheetName === sheetName);
-    const additionalDetailsErrors: any = [];
 
+    var errorData = request.body.sheetErrorDetails.filter((error: any) => error.sheetName === sheetName);
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)